<!DOCTYPE html>
<html xmlns="http://www.w3.org/1999/xhtml" lang="en" xml:lang="en"><head>

<meta charset="utf-8">
<meta name="generator" content="quarto-1.3.336">

<meta name="viewport" content="width=device-width, initial-scale=1.0, user-scalable=yes">


<title>cv_en</title>
<style>
code{white-space: pre-wrap;}
span.smallcaps{font-variant: small-caps;}
div.columns{display: flex; gap: min(4vw, 1.5em);}
div.column{flex: auto; overflow-x: auto;}
div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
ul.task-list{list-style: none;}
ul.task-list li input[type="checkbox"] {
  width: 0.8em;
  margin: 0 0.8em 0.2em -1em; /* quarto-specific, see https://github.com/quarto-dev/quarto-cli/issues/4556 */ 
  vertical-align: middle;
}
</style>


<script src="site_libs/quarto-nav/quarto-nav.js"></script>
<script src="site_libs/quarto-nav/headroom.min.js"></script>
<script src="site_libs/clipboard/clipboard.min.js"></script>
<script src="site_libs/quarto-search/autocomplete.umd.js"></script>
<script src="site_libs/quarto-search/fuse.min.js"></script>
<script src="site_libs/quarto-search/quarto-search.js"></script>
<meta name="quarto:offset" content="./">
<script src="site_libs/quarto-html/quarto.js"></script>
<script src="site_libs/quarto-html/popper.min.js"></script>
<script src="site_libs/quarto-html/tippy.umd.min.js"></script>
<script src="site_libs/quarto-html/anchor.min.js"></script>
<link href="site_libs/quarto-html/tippy.css" rel="stylesheet">
<link href="site_libs/quarto-html/quarto-syntax-highlighting.css" rel="stylesheet" class="quarto-color-scheme" id="quarto-text-highlighting-styles">
<link href="site_libs/quarto-html/quarto-syntax-highlighting-dark.css" rel="prefetch" class="quarto-color-scheme quarto-color-alternate" id="quarto-text-highlighting-styles">
<script src="site_libs/bootstrap/bootstrap.min.js"></script>
<link href="site_libs/bootstrap/bootstrap-icons.css" rel="stylesheet">
<link href="site_libs/bootstrap/bootstrap.min.css" rel="stylesheet" class="quarto-color-scheme" id="quarto-bootstrap" data-mode="light">
<link href="site_libs/bootstrap/bootstrap-dark.min.css" rel="prefetch" class="quarto-color-scheme quarto-color-alternate" id="quarto-bootstrap" data-mode="dark">
<script id="quarto-search-options" type="application/json">{
  "location": "navbar",
  "copy-button": false,
  "collapse-after": 3,
  "panel-placement": "end",
  "type": "overlay",
  "limit": 20,
  "language": {
    "search-no-results-text": "No results",
    "search-matching-documents-text": "matching documents",
    "search-copy-link-title": "Copy link to search",
    "search-hide-matches-text": "Hide additional matches",
    "search-more-match-text": "more match in this document",
    "search-more-matches-text": "more matches in this document",
    "search-clear-button-title": "Clear",
    "search-detached-cancel-button-title": "Cancel",
    "search-submit-button-title": "Submit"
  }
}</script>


<link rel="stylesheet" href="styles.css">
</head>

<body class="nav-fixed">

<div id="quarto-search-results"></div>
  <header id="quarto-header" class="headroom fixed-top">
    <nav class="navbar navbar-expand-lg navbar-dark ">
      <div class="navbar-container container-fluid">
            <div id="quarto-search" class="" title="Search"></div>
          <button class="navbar-toggler" type="button" data-bs-toggle="collapse" data-bs-target="#navbarCollapse" aria-controls="navbarCollapse" aria-expanded="false" aria-label="Toggle navigation" onclick="if (window.quartoToggleHeadroom) { window.quartoToggleHeadroom(); }">
  <span class="navbar-toggler-icon"></span>
</button>
          <div class="collapse navbar-collapse" id="navbarCollapse">
            <ul class="navbar-nav navbar-nav-scroll me-auto">
  <li class="nav-item">
    <a class="nav-link" href="./index.html" rel="" target="">
 <span class="menu-text">🇬🇧 Curriculum Vitae in English</span></a>
  </li>  
  <li class="nav-item">
    <a class="nav-link" href="./cv_hu.html" rel="" target="">
 <span class="menu-text">🇭🇺 Curriculum Vitae in Hunagrian</span></a>
  </li>  
</ul>
            <div class="quarto-navbar-tools ms-auto">
  <a href="" class="quarto-color-scheme-toggle quarto-navigation-tool  px-1" onclick="window.quartoToggleColorScheme(); return false;" title="Toggle dark mode"><i class="bi"></i></a>
</div>
          </div> <!-- /navcollapse -->
      </div> <!-- /container-fluid -->
    </nav>
</header>
<!-- content -->
<div id="quarto-content" class="quarto-container page-columns page-rows-contents page-layout-article page-navbar">
<!-- sidebar -->
<!-- margin-sidebar -->
    <div id="quarto-margin-sidebar" class="sidebar margin-sidebar">
        <nav id="TOC" role="doc-toc" class="toc-active">
    <h2 id="toc-title">On this page</h2>
   
  <ul class="collapse">
  <li><a href="#curriculum-vitae-of-eszter-ari" id="toc-curriculum-vitae-of-eszter-ari" class="nav-link active" data-scroll-target="#curriculum-vitae-of-eszter-ari"><em>Curriculum Vitae</em> of Eszter Ari</a></li>
  <li><a href="#positions" id="toc-positions" class="nav-link" data-scroll-target="#positions">Positions</a>
  <ul class="collapse">
  <li><a href="#present" id="toc-present" class="nav-link" data-scroll-target="#present">Present</a></li>
  <li><a href="#formerly" id="toc-formerly" class="nav-link" data-scroll-target="#formerly">Formerly</a></li>
  </ul></li>
  <li><a href="#degrees-dipliomas" id="toc-degrees-dipliomas" class="nav-link" data-scroll-target="#degrees-dipliomas">Degrees &amp; Dipliomas</a></li>
  <li><a href="#awards-scholarships" id="toc-awards-scholarships" class="nav-link" data-scroll-target="#awards-scholarships">Awards &amp; Scholarships</a></li>
  <li><a href="#grants" id="toc-grants" class="nav-link" data-scroll-target="#grants">Grants</a></li>
  <li><a href="#teaching-activities" id="toc-teaching-activities" class="nav-link" data-scroll-target="#teaching-activities">Teaching Activities</a>
  <ul class="collapse">
  <li><a href="#present-1" id="toc-present-1" class="nav-link" data-scroll-target="#present-1">Present</a></li>
  <li><a href="#formerly-1" id="toc-formerly-1" class="nav-link" data-scroll-target="#formerly-1">Formerly</a></li>
  </ul></li>
  <li><a href="#supervising-metoring" id="toc-supervising-metoring" class="nav-link" data-scroll-target="#supervising-metoring">Supervising &amp; Metoring</a></li>
  <li><a href="#main-workshops" id="toc-main-workshops" class="nav-link" data-scroll-target="#main-workshops">Main Workshops</a>
  <ul class="collapse">
  <li><a href="#as-lecturer" id="toc-as-lecturer" class="nav-link" data-scroll-target="#as-lecturer">As lecturer</a></li>
  <li><a href="#as-participant" id="toc-as-participant" class="nav-link" data-scroll-target="#as-participant">As participant</a></li>
  </ul></li>
  <li><a href="#memberships" id="toc-memberships" class="nav-link" data-scroll-target="#memberships">Memberships</a></li>
  <li><a href="#languages" id="toc-languages" class="nav-link" data-scroll-target="#languages">Languages</a></li>
  <li><a href="#areas-of-interest" id="toc-areas-of-interest" class="nav-link" data-scroll-target="#areas-of-interest">Areas of Interest</a></li>
  <li><a href="#professional-skills" id="toc-professional-skills" class="nav-link" data-scroll-target="#professional-skills">Professional Skills</a></li>
  <li><a href="#editing-reviewing" id="toc-editing-reviewing" class="nav-link" data-scroll-target="#editing-reviewing">Editing &amp; Reviewing</a></li>
  <li><a href="#popularising-science" id="toc-popularising-science" class="nav-link" data-scroll-target="#popularising-science">Popularising Science</a></li>
  <li><a href="#scientific-websites" id="toc-scientific-websites" class="nav-link" data-scroll-target="#scientific-websites">Scientific Websites</a></li>
  <li><a href="#publications" id="toc-publications" class="nav-link" data-scroll-target="#publications">Publications</a>
  <ul class="collapse">
  <li><a href="#preprints" id="toc-preprints" class="nav-link" data-scroll-target="#preprints">Preprints</a></li>
  <li><a href="#published" id="toc-published" class="nav-link" data-scroll-target="#published">Published</a></li>
  </ul></li>
  <li><a href="#oral-presentations-in-conferences" id="toc-oral-presentations-in-conferences" class="nav-link" data-scroll-target="#oral-presentations-in-conferences"><strong>ORAL PRESENTATIONS IN CONFERENCES</strong></a></li>
  <li><a href="#other-invited-talks" id="toc-other-invited-talks" class="nav-link" data-scroll-target="#other-invited-talks"><strong>OTHER INVITED TALKS</strong></a></li>
  <li><a href="#poster-presentations-in-conferences" id="toc-poster-presentations-in-conferences" class="nav-link" data-scroll-target="#poster-presentations-in-conferences"><strong>POSTER PRESENTATIONS IN CONFERENCES</strong></a></li>
  </ul>
<div class="quarto-alternate-formats"><h2>Other Formats</h2><ul><li><a href="cv_en.pdf"><i class="bi bi-file-pdf"></i>PDF</a></li><li><a href="cv_en.docx"><i class="bi bi-file-word"></i>MS Word</a></li></ul></div></nav>
    </div>
<!-- main -->
<main class="content" id="quarto-document-content">



<section id="curriculum-vitae-of-eszter-ari" class="level1">
<h1><em>Curriculum Vitae</em> of Eszter Ari</h1>
<p>Bioinformatician Researcher, University Teacher</p>
<p><a href="mailto:arieszter@gmail.com">arieszter@gmail.com</a></p>
<p>Websites:</p>
<ul>
<li>Group website: <a href="https://genet.elte.hu/bioinformatics">Bioinformatics Research Group at Eötvös Loránd University (ELTE), Budapest, H</a></li>
<li>Personal website at <a href="https://genet.elte.hu/en/content/ari-eszter.t.16148">ELTE, Budapest, H</a></li>
<li>Personal website at <a href="http://group.szbk.u-szeged.hu/sysbiol/papp-balazs-lab-member.html#eszter-ari">Biological Research Centre, Szeged, H</a></li>
<li>Public pages: <img src="images/ResearchGate_icon_SVG.png" class="img-fluid" width="24" alt="Researchgate"> <img src="images/ORCID_iD_32x32.svg.png" class="img-fluid" width="24" alt="ORCiD"> <img src="images/web_of_science.png" class="img-fluid" width="22" alt="ResearcherID"> <img src="images/scopus.png" class="img-fluid" width="24" alt="Scopus"> <img src="images/mtmt.jpg" class="img-fluid" width="24" alt="MTMT"> <img src="images/twitter_x_new_logo_square_x_icon.png" class="img-fluid" width="24" alt="X (Twitter)"> <img src="images/Bluesky_Logo.svg.png" class="img-fluid" width="27" alt="BlueSky"></li>
</ul>
</section>
<section id="positions" class="level1">
<h1>Positions</h1>
<section id="present" class="level2">
<h2 class="anchored" data-anchor-id="present">Present</h2>
<p><strong>Associate professor –</strong> <strong>Eötvös Loránd University (ELTE), Department of Genetics</strong></p>
<blockquote class="blockquote">
<p>H-1117, Pázmány Péter stny 1/C, Budapest, Hungary</p>
</blockquote>
<ul>
<li>2025 - PRESENT Teaching various bioinformatics courses to biology master and graduate students. Supervising undergraduate and graduate students. Coordinating the Molecular Genetics, Cell- and Developmental Biology specialisation.</li>
<li>2019 - 2025 assistant professor</li>
<li>2009 - 2019 assistant lecturer</li>
<li>2007 - 2009 scientific research associate</li>
</ul>
<p><strong>Research fellow – HUN-REN</strong><a href="#fn1" class="footnote-ref" id="fnref1" role="doc-noteref"><sup>1</sup></a> <strong>Biological Research Centre (BRC), Institute of Biochemistry, Szeged</strong></p>
<blockquote class="blockquote">
<p>H-6726, Temesvári krt. 62, Szeged, Hungary</p>
</blockquote>
<ul>
<li>2016 - PRESENT Investigating the evolution of antibiotic resistance and virulence using phylogenetic and comparative genomics approaches in Balázs Papp’s Lab, from 2020 as a project leader.</li>
</ul>
<p><strong>Research fellow – Hungarian Centre of Excellence for Molecular Medicine (HCEMM), Metabolic Systems Biology Research Group, Szeged</strong></p>
<ul>
<li>2019 - PRESENT</li>
</ul>
</section>
<section id="formerly" class="level2">
<h2 class="anchored" data-anchor-id="formerly">Formerly</h2>
<p><strong>Postdoctoral researcher – Univ. of Veterinary Medicine (Vet-Med Uni), Institut für Populationsgenetik, Vienna, A</strong></p>
<ul>
<li>2012 - 2014 Analysing the RNA-seq data of thermal adapted fruit fly populations in Christian Schlötterer’s Lab.</li>
</ul>
</section>
</section>
<section id="degrees-dipliomas" class="level1">
<h1>Degrees &amp; Dipliomas</h1>
<ul>
<li><strong>Habilitation</strong>: ELTE, Budapest, H, 2023</li>
<li><strong>PhD</strong>: Biology Doctoral School, Theoretical and Evolutionary Biology Doctoral Programme, ELTE, Budapest, H, 2012</li>
<li><strong>Bachelor and Master</strong>: Majoring in Applied Zoology, Faculty of Veterinary Sciences<a href="#fn2" class="footnote-ref" id="fnref2" role="doc-noteref"><sup>2</sup></a>, Szent István University, Budapest, H, 1999 - 2004</li>
</ul>
</section>
<section id="awards-scholarships" class="level1">
<h1>Awards &amp; Scholarships</h1>
<ul>
<li>Outstanding scientific publication award from Excellence Fund of Eötvös Loránd University, 2024 and 2022, Budapest, H</li>
<li>Junior fellowship at Collegium Budapest – Institute for Advanced Study, 2009, Budapest, H</li>
<li>Scholarship of the Fac. Veterinary Sci., Szent István Univ., 2003 - 2004, Budapest, H</li>
<li>2nd Prize of the Conference for Student Scientists at Fac. Veterinary Sci., Szent István Univ., 2003, Budapest, H</li>
</ul>
</section>
<section id="grants" class="level1">
<h1>Grants</h1>
<ul>
<li><strong>Supported Research Groups Programme 2025-2028</strong>: Role: co-applicant; Title: Genomic surveillance for precision therapies against antibiotic-resistant bacteria; Lead researcher: Papp Balázs; Number: TKCS-2024/66; amount awarded to ELTE: 30,000,000 HUF</li>
<li><strong>Hungarian National Research Fund Grant – Postdoctoral Excellence Programme</strong> <strong>2020 - 2023</strong>: Role: Principal investigator; Title: How do ‘superbugs’ emerge? - A systematic study of the mobility of resistance and virulence genes in human microbiota and pathogenic bacteria; Nr. <a href="http://nyilvanos.otka-palyazat.hu/index.php?menuid=930&amp;num=131839&amp;lang=EN">131839</a>; 25,500,000 HUF</li>
<li><strong>Promoting Excellence in Education, ELTE 2020</strong>: Role: PI; Title: Development of the infrastructure of the computer practical room; 4,000,000 HUF</li>
<li><strong>Visegrad 4 Eastern Partnership (V4Eap) 2014 - 2015</strong>; Role: Supervisor; Connected to the fellowship of Zeljko Popovic; Title: DORMANCYbase – developing a database on gene and protein expression during dormancy in animals; 3,000 EUR</li>
</ul>
</section>
<section id="teaching-activities" class="level1">
<h1>Teaching Activities</h1>
<section id="present-1" class="level2">
<h2 class="anchored" data-anchor-id="present-1">Present</h2>
<ul>
<li><a href="https://docs.google.com/document/d/1pBIYs0gVawKaT1g_Sf9lYj4QCJCE76MJdwDohb6FXFI/edit?usp=drive_link">Bioinformatics</a> (L, P; EN) – for Master students</li>
<li><a href="https://docs.google.com/document/d/e/2PACX-1vTyT853D-UqTq1E__rH6UFsIQ1j-EEWk2wDxnHNcBCRw4pA6JemHEv2ND_e66paI_Zr5XYahY133T7e/pub">Advanced R programming</a> (P; EN) – for Master students</li>
<li><a href="https://docs.google.com/document/d/1okHYnyxcyyCyohYHSyNDFSWE4sEnllHoQKyvVrJ2-Z0/edit?usp=sharing">Analysis of Omics data</a> (P; EN) – for Master students</li>
<li>Bioinformatic seminars (L; EN) – for PhD students</li>
</ul>
</section>
<section id="formerly-1" class="level2">
<h2 class="anchored" data-anchor-id="formerly-1">Formerly</h2>
<ul>
<li>Introduction to Bioinformatics (L; HU) – for Bachelor students</li>
<li>Genetic lab. practices (selected P; EN, HU) – for Bachelor students</li>
<li>Genomics (selected L; EN, HU) – for Master students</li>
<li>Molecular evolution (selected L; HU) – for Master students</li>
<li>Evolutionary biology (selected L; HU) – for Master students</li>
<li>Discrete mathematical methods in biology (L, P; HU) – for Master and PhD students</li>
</ul>
</section>
</section>
<section id="supervising-metoring" class="level1">
<h1>Supervising &amp; Metoring</h1>
<ul>
<li><a href="https://doktori.hu/index.php?menuid=192&amp;lang=EN&amp;sz_ID=11414">The personal site at the website of Hungarian Doctoral Council</a></li>
<li><strong>Graduated students</strong>: 2 co-supervised PhD, 6 Master, 12 Bachelor, and 3 Scientific Student Association students</li>
<li><strong>At present</strong>: Co-supervising 4 PhD students, supervising 2 Master, 2 Bachelor, and 3 Scientific Student Association students</li>
<li><strong>Visiting student:</strong> 1 (with Support for summer internships for Hungarian students studying abroad grant)</li>
</ul>
</section>
<section id="main-workshops" class="level1">
<h1>Main Workshops</h1>
<section id="as-lecturer" class="level2">
<h2 class="anchored" data-anchor-id="as-lecturer">As lecturer</h2>
<ul>
<li><em>ELTE Carpathian Basin Hungarian Summer University</em>, 2022: Genetic analysis of the virus variants determining the COVID epidemic in Hungary</li>
<li><a href="https://www.popgen-vienna.at/training/other-courses/ngs-workshop-2014/"><em>Practical Course on Next Generation Sequencing for Population Genetics and Experimental Evolution</em></a>, 2014 Vet-Med Uni Vienna, A: Practicals about RNA-seq data analysis</li>
</ul>
</section>
<section id="as-participant" class="level2">
<h2 class="anchored" data-anchor-id="as-participant">As participant</h2>
<ul>
<li><em>EMBO Training: Laboratory Leadership Course</em>, 2024, Budapest, H</li>
<li><em>An R Reproducibility Toolkit for the practical researcher</em>, 2022, online</li>
<li><em>8th International Course in Microbial Ecology – Microbiome Metagenome Analysis</em>, 2017, CNR-ISE, Verbania, I</li>
<li><em>EMBO course: Bioinformatics and Comparative Genome Analyses</em>, 2011, Inst. Pasteur Paris, F</li>
<li><em>Computational Phyloinformatics Course</em>, 2008, Duke Univ., Durham, NC, USA</li>
<li><em>Workshop on Molecular Evolution</em>, 2006, Marine Biological Lab., Woods Hole, MA, USA</li>
</ul>
</section>
</section>
<section id="memberships" class="level1">
<h1>Memberships</h1>
<ul>
<li><a href="http://mabit.ttk.mta.hu/doc/en/home">Hungarian Society for Bioinformatics</a>, 2016 – a member of the directory board; 2006 – member</li>
<li>Hungarian Academy of Sciences, Interdepartmental Scientific Committee on Bioinformatics, 2021 – committee member</li>
<li>Hungarian node of the <a href="https://elixir-europe.org/about-us/who-we-are/nodes/hungary">Elixir Europe</a>, 2023 – lead researcher</li>
<li><a href="https://scicomp.hu/?lang=en">Institute of Scientific Computing, Society</a> (SciComp) membership, 2024</li>
<li>Hungarian Academy of Sciences, 2013 – a member of the public body (ID: 31344)</li>
</ul>
</section>
<section id="languages" class="level1">
<h1>Languages</h1>
<ul>
<li>Hungarian – mother tongue</li>
<li>English – C1 level</li>
<li>Italian – A1 level</li>
</ul>
</section>
<section id="areas-of-interest" class="level1">
<h1>Areas of Interest</h1>
<ul>
<li>Bioinformatics</li>
<li>Evolutionary Biology</li>
<li>Antibiotic Resistance</li>
<li>Comparative genomics</li>
<li>Molecular phylogenetics</li>
<li>Metagenomics</li>
<li>Transcriptomics</li>
<li>Transcriptional regulation</li>
<li>Data analysis</li>
</ul>
</section>
<section id="professional-skills" class="level1">
<h1>Professional Skills</h1>
<ul>
<li>Developing and using different bioinformatic methods, software and pipelines.</li>
<li><em>R</em> programming language – professional level</li>
<li><em>Linux Shell</em> scripting – intermediate level</li>
</ul>
</section>
<section id="editing-reviewing" class="level1">
<h1>Editing &amp; Reviewing</h1>
<ul>
<li><a href="https://www.frontiersin.org/research-topics/62823/gene-evolution-and-mutation-in-evolutionary-systems-biology">Topic editor in Frontiers in Systems Biology</a></li>
<li>Reviewer in journals: <em>Brief. Bioinformatics</em> (3); <em>Evol. Biol.</em> (1); <em>Genome Biol. Evol.</em> (1); <em>J. Math. Chem.</em> (4); <em>Mol. Ecol.</em> (1); <em>Mol. Phylogenetics Evol.</em> (1); <em>Sci. Rep.</em> (1); <em>Opusc. Zool.</em> (1); <em>Plos One</em> (2)</li>
<li>Reviewing: PhD Theses (4), Grant proposals: Hungarian Scientific Research Fund Grant (3)</li>
</ul>
</section>
<section id="popularising-science" class="level1">
<h1>Popularising Science</h1>
<ul>
<li>Presenting at the Festival of ELTE: Global map of evolutionary dependencies between antibiotic resistance and virulence genes in <em>E. coli</em>. 2022, Budapest, H</li>
<li>Creating a website for The genomic epidemiology of SARS-CoV-2 in Hungary</li>
<li>Presenting at Capital of Science “Tudományok Fővárosa”: The genomic epidemiology of SARS-CoV-2 in Hungary, 2020, Budapest, H</li>
<li>Presenting at Night of Scientists “Kutatók Éjszakája”, 2018 and 2020</li>
</ul>
</section>
<section id="scientific-websites" class="level1">
<h1>Scientific Websites</h1>
<ul>
<li>Supervising the creation of the <a href="https://tflink.net/">TFLink</a>, a transcription factor - target gene database.</li>
<li>Supervising the creation of the <a href="https://cran.r-project.org/web/packages/mulea/">mulea</a> and the <a href="https://bioconductor.org/packages/release/data/experiment/html/muleaData.html">muleaData</a> R packages for functional enrichment analysis and dataset.</li>
</ul>
</section>
<section id="publications" class="level1">
<h1>Publications</h1>
<p><a href="https://scholar.google.com/citations?user=cYCKyr0AAAAJ"><img src="images/Google_Scholar_logo.svg.png" class="img-fluid" alt="Google Scholar" width="24"></a> <a href="https://m2.mtmt.hu/api/publication?cond=published;eq;true&amp;cond=core;eq;true&amp;cond=authors.mtid;eq;10020542&amp;sort=publishedYear,desc&amp;sort=firstAuthor,asc&amp;size=20&amp;fields=template,citations:0,citationCount&amp;cite_type=3&amp;&amp;page=1&amp;labelLang=eng"><img src="images/mtmt.jpg" class="img-fluid" width="24" alt="MTMT"></a></p>
<ul>
<li>Nr. of published papers: 27 (D1: 17 + Q1: 5)</li>
<li>Nr. of first, last or corresponding author publications: <strong>8</strong>, Cumulative impact factor of these publications: <strong>47.22</strong></li>
<li>Cumulative impact factor of all publications (IF): <strong>218.56</strong></li>
<li>Independent citations of all publications (IC): <strong>543</strong></li>
<li>h-index: <strong>14</strong></li>
</ul>
<section id="preprints" class="level2">
<h2 class="anchored" data-anchor-id="preprints">Preprints</h2>
<p>Balogh GM, Koncz B, Asztalos L, <strong>Ari E</strong>, Papp B, Szebeni GJ, Gémes N, Pál C &amp; Manczinger M (2024) APOBEC3-associated mutations generate immunogenic peptides in SARS-CoV-2. <em>Nature Communications</em>, under review</p>
</section>
<section id="published" class="level2">
<h2 class="anchored" data-anchor-id="published">Published</h2>
<p>Daruka L, Czikkely MS, Szili P, Farkas Z, Balogh D, Maharramov E, Vu TH, Sipos L, Vincze BD, Grézal G, Juhász Sz, Dunai A, Daraba A, Számel M, Sári T, Stirling T, Vásárhelyi BM, <strong>Ari E</strong>, Christodoulou C, Manczinger M, Enyedi MZs, Jaksa G, van Houte S, Pursey E, Papp CG, Szilovics Z, Pintér L, Haracska L, Gácser A, Kintses B, Papp B, Pál Cs (2024) ESKAPE pathogens rapidly develop resistance against antibiotics in development. <em>Nature Microbiology</em>, accepted (preprint: bioRxiv, 2023, <a href="https://doi.org/10.1101/2023.07.23.550022">2023.07.23.550022</a>) <strong>D1, IF: 20.5, IC: 3</strong></p>
<p>Turek C, Ölbei M, Stirling T, Fekete G, Tasnádi E, Gul L, Bohár B, Papp B, Jurkowski W &amp; <strong>Ari E*</strong> (2024) <a href="https://doi.org/10.1186/s12859-024-05948-7">mulea - an R package for enrichment analysis using various ontologies and empirical false discovery rate</a>. BMC Bioinformatics, 25(1): 334. <strong>Q1, IF: 2.9</strong> <strong>(*corresponding author)</strong></p>
<p>Koncz M, Stirling T, Mehdi HH, Méhi O, Eszenyi B, Asbóth A, Apjok G, Tóth Á, Orosz L, Vásárhelyi BM, <strong>Ari E</strong>, Daruka L, Polgár TF, Schneider Gy, Zalokh SA, Számel M, Fekete G, Bohár B, Nagy Varga K, Visnyovszki Á, Székely E, Licker MS, Izmendi O, Costache C, Gajic I, Lukovic B, Molnár Sz, Szőcs-Gazdi UO, Bozai Cs, Indreas M, Kristóf K, Van der Henst C, Breine A; Pál C, Papp B &amp; Kintses B (2024) <a href="https://www.cell.com/cell/fulltext/S0092-8674(24)01027-4">Genomic surveillance as a scalable framework for precision phage therapy against antibiotic-resistant pathogens</a>. Cell, 187(21) 5901 - 5918.e28, <strong>D1, IF: 45.5, IC: 1</strong></p>
<p>Gerber D, Szeifert B, Székely O, Egyed B, Gyuris B, Giblin JI, Horváth A, Köhler K, Kulcsár G, Kustár Á, Major I, Molnár M, Palcsu L, Szeverényi V, Fábián S, Mende BG, Bondár M, <strong>Ari E*</strong>, Kiss V* &amp; Szécsényi-Nagy A* (2023) <a href="https://academic.oup.com/mbe/advance-article/doi/10.1093/molbev/msad182/7240678">Interdisciplinary analyses of Bronze Age communities from Western Hungary reveal complex population histories.</a> Molecular Biology and Evolution, 40(9): msad182. <strong>D1, IF: 11.0, IC: 6 (*shared corresponding authorship)</strong></p>
<p>Bacsur P, Rutka M, Resál T, Szántó K, Jójárt B, Bálint A, <strong>Ari E</strong>, Walliyulah A, Kintses B, Fehér T, Asbóth A, Pigniczki D, Bor R, Fábián A, Maléth J, Szepes Z, Farkas K &amp; Molnár T (2023) <a href="https://journals.sagepub.com/doi/full/10.1177/17562848231174298">Effects of bowel cleansing on the composition of the gut microbiome in inflammatory bowel disease patients and healthy controls</a>. Therapeutic Advances in Gastroenterology, 16: 1-13. <strong>Q1, IF: 3.9, IC: 3</strong></p>
<p>Sturm Á, Saskői É, Hotzi B, Tarnóci A, Barna J, Bodnár F, Sharma H, Kovács T, <strong>Ari E</strong>, Weinhardt N, Kerepesi C, Perczel A, Ivics Z &amp; Vellai T (2023) <a href="https://doi.org/10.1038/s41467-023-40957-9">Downregulation of transposable elements extends lifespan in Caenorhabditis elegans.</a> Nature Communications, 14(1): 5278. <strong>D1, IF:&nbsp;17.7, IC: 10</strong></p>
<p><strong>Ari E</strong>, Vásárhelyi BM, Kemenesi G, Tóth GE, Zana B, Somogyi B, Lanszki Z, Röst G, Jakab F,&nbsp; Papp B &amp; Kintses B (2022) <a href="https://doi.org/10.1093/ve/veac069">A single early introduction governed viral diversity in the second wave of SARS-CoV-2 epidemic in Hungary</a>. Virus Evolution, 8(2): veac069. <strong>D1, IF: 5.3, IC: 3</strong></p>
<p>Csabai L, Fazekas D, Kadlecsik T, Szalay-Bekő M, Bohár B, Madgwick M, Módos D, Ölbei M, Gul L, Sudhakar P, Kubisch J, Oyeyemi OJ, Liska O, <strong>Ari E</strong>, Hotzi B, Billes VA, Molnár E, Földvári-Nagy L, Csályi K, Demeter A, Pápai N, Koltai M, Varga M, Lenti K, Farkas IJ, Türei D, Csermely P, Vellai T &amp; Korcsmáros T (2022) <a href="https://academic.oup.com/nar/advance-article/doi/10.1093/nar/gkab909/6389625">SignaLink3: A multi-layered resource to uncover tissue-specific signaling networks</a>. Nucleic Acids Research, 50(D1): 701-709. <strong>D1, IF: 14.9, IC: 22</strong></p>
<p>Liska O, Bohár B, Hidas H, Korcsmáros T, Papp B, Fazekas D &amp; <strong>Ari E*</strong> (2022) <a href="https://doi.org/10.1093/database/baac083">TFLink: An integrated gateway to access transcription factor - target gene interactions for multiple species</a>. Database, 2022, baac083. <strong>“High impact paper” according to <a href="http://scientometrics.org">scientometrics.org</a>.</strong> <strong>D1, IF: 5.8, IC: 62 (*corresponding author)</strong></p>
<p>Bálint A, Farkas K, Kintses B, Méhi O, Vásárhelyi BM, <strong>Ari E</strong>, Pál C, Madácsy T, Maléth J, Szántó KJ, Nagy I, Rutka M, Bacsur P, Szűcs D, Szepes Z, Nagy F, Fábián A, Bor R, Milassin Á &amp; Molnár T (2020) <a href="https://www.mdpi.com/1424-8247/13/11/346/htm">Functional anatomical changes in ulcerative colitis patients determine their gut microbiota composition and consequently the possible treatment outcome</a>. Pharmaceuticals, 13(11): 346-362. <strong>D1, IF: 5.68, IC: 17</strong></p>
<p>Kintses B, Jangir PK, Fekete G, Számel M, Méhi O, Spohn R, Daruka L, Martins A, Hosseinnia A, Gagarinova A, Kim S, Phanse S, Csörgö B, Györkei Á, <strong>Ari E</strong>, Lázár V, Faragó A, Nagy I, Babu M, Pál C &amp; Papp B (2019) <a href="https://www.nature.com/articles/s41467-019-13618-z">Chemical-genetic profiling reveals limited cross-resistance between antimicrobial peptides with different modes of action</a>. Nature Communications, 10: 5731. <strong>D1, IF: 12.12, IC: 33</strong></p>
<p>Kintses B*, Méhi O*, <strong>Ari E*</strong>, Számel M, Györkei Á, Jangir PK, Nagy I, Pál F, Fekete G, Tengölics R, Nyerges Á, Likó I, Bálint A, Molnár T, Bálint B, Vásárhelyi BM, Bustamante M, Papp B &amp; Pál C (2019) <a href="https://www.nature.com/articles/s41564-018-0313-5">Phylogenetic barriers to horizontal transfer of antimicrobial peptide resistance genes in the human gut microbiota</a>. Nature Microbiology, 4(3): 447-458. <strong>Craig MacLean recommended it in <a href="https://facultyopinions.com/prime/734649950#eval793557383">F1000Prime</a> as being of special significance in its field. It was selected as a part of the <a href="https://www.nature.com/collections/gjdhfgjiid">Nature Gut Microbiota core collection</a>.</strong> <strong>D1, IF:&nbsp;15.54, IC: 74 (*shared first authorship)</strong></p>
<p>Sudhakar P, Claire-Jacomin A, Hautefort I, Samavedam S, Fatemian K, <strong>Ari E</strong>, Gul L, Demeter A, Jones E,&nbsp; Korcsmaros T &amp; Nezis JP (2019) <a href="https://www.tandfonline.com/doi/full/10.1080/15548627.2019.1590519">Targeted interplay between bacterial pathogens and host autophagy</a>. Autophagy, 15(9): 1620-1633. <strong>D1, IF: 11.06, IC: 40</strong></p>
<p>Sun D, Ren X, <strong>Ari E</strong>, Korcsmaros T, Csermely P &amp; Wu L-Y (2019) <a href="https://academic.oup.com/bib/article-abstract/20/1/89/4079498?redirectedFrom=fulltext">Discovering cooperative biomarkers for heterogeneous complex disease diagnoses</a>. Briefings in Bioinformatics, 20(1): 89-101. <strong>D1, IF: 9.10, IC: 14</strong></p>
<p>Nyerges Á, Csörgő B, Draskovits G, Kintses B, Szili P, Ferenc Gy, Révész T, <strong>Ari E</strong>, Nagy I, Bálint B, Vásárhelyi BM, Bihari P, Számel M, Balogh D, Papp H, Kalapis D, Papp B &amp; Pál C (2018) <a href="https://www.pnas.org/content/115/25/E5726">Directed evolution of multiple genomic loci allows the prediction of antibiotic resistance</a>. Proceedings of the National Academy of Sciences of the United States of America, 115(25): E5726-E5735. <strong>D1, IF: 9.58, IC: 73</strong></p>
<p>Fodor E, Sigmond T, <strong>Ari E</strong>, Lengyel K, Takács-Vellai K, Varga M &amp; Vellai T (2017) <a href="https://www.sciencedirect.com/science/article/pii/S0076687916303949">Methods to study autophagy in zebrafish</a>. Methods in Enzymology 588: 467-96. <strong>Q1, IF: 1.48, IC: 19</strong></p>
<p>Métris A, Sudhakar P, Fazekas D, Demeter A, <strong>Ari E</strong>, Ölbei M, Branchu P, Kingsley RA, Baranyi J &amp; Korcsmáros T (2017) <a href="https://www.nature.com/articles/s41540-017-0034-z">SalmoNet, an integrated network of ten Salmonella enterica strains reveals common and distinct pathways to host adaptation</a>. Npj Systems Biology and Applications 3(1): s41540-17-34-NaN-17. <strong>Q1, IF: 3.81, IC: 27</strong></p>
<p>Ricotta C, <strong>Ari E</strong>, Bonanomi G, Giannino F, Heathfield D, Mazzoleni S &amp; Podani J (2017) <a href="https://akjournals.com/view/journals/168/18/1/article-p37.xml">Spatial analysis of phylogenetic community structure: New version of a classical method</a>. Community Ecology. 18(1): 37-46. (preprint: 2015, PeerJ PrePrints e1145) <strong>Q2, IF: 0.98, IC: 2</strong></p>
<p>Ács É, <strong>Ari E</strong>, Duleba M, Dressler M, Genkal SI, Jakó É, Rimet F, Ector L &amp; Kiss KT (2016) <a href="https://fottea.czechphycology.cz/artkey/fot-201601-0006_pantocsekiella_a_new_centric_diatom_genus_based_on_morphological_and_genetic_studies.php">Pantocsekiella, a new centric diatom genus based on morphological and genetic studies</a>. Fottea 16(1): 56-78. <strong>Q2, IF: 0.6, IC: 50</strong></p>
<p><strong>Ari E*</strong> &amp; Jakó É (2016) <a href="https://onlinelibrary.wiley.com/doi/abs/10.1002/cem.2781">Classification of tRNA isoacceptor sequences by using graph-based molecular descriptors</a>. Journal of Chemometrics 30(1): 182-187. <strong>Q2, IF: 2.03 (*corresponding author)</strong></p>
<p>Módos D, Brooks J, Fazekas D, <strong>Ari E</strong>, Vellai T, Csermely P, Korcsmáros T &amp; Lenti K (2016) <a href="https://www.nature.com/articles/srep38588">Identification of critical paralog groups with indispensable roles in the regulation of signaling flow</a>. Scientific Reports 6: 38588. <strong>D1, IF: 4.62, IC: 6</strong></p>
<p>Földvári-Nagy L, <strong>Ari E</strong>, Csermely P, Korcsmáros T &amp; Vellai T (2014) <a href="https://www.nature.com/articles/srep05829">Starvation-response may not involve Atg1-dependent autophagy induction in non-unikont parasites</a>. Scientific Reports 4: 5829. <strong>D1, IF: 5.85, IC: 16</strong></p>
<p><strong>Ari E*</strong>, Ittzés P, Podani J, Le Thi QC &amp; Jakó É (2012) <a href="https://www.sciencedirect.com/science/article/abs/pii/S1055790312000279">Comparison of Boolean analysis and standard phylogenetic methods using artificially evolved and natural mt-tRNA sequences from great apes</a>. Molecular Phylogenetics and Evolution 63(1): 193-202. <strong>D1, IF: 4.42, IC: 5 (*corresponding author)</strong></p>
<p>Boros G, Cech G, <strong>Ari E</strong> &amp; Dózsa-Farkas K (2010) <a href="https://www.tandfonline.com/doi/abs/10.1080/09397140.2010.10638454">Extension of employing ITS region in the investigation of Hungarian Fridericia species (Oligochaeta: Enchytraeidae)</a>. Zoology in the Middle East 51(sup2): 23-30. <strong>Q3, IF: 0.3, IC: 2</strong></p>
<p>Tihanyi B, Vellai T, Regős Á, <strong>Ari E</strong>, Müller F &amp; Takács-Vellai K (2010) <a href="https://bmcdevbiol.biomedcentral.com/articles/10.1186/1471-213X-10-78">The C. elegans Hox gene ceh-13 regulates cell migration and fusion in a non-colinear way. Implications for the early evolution of Hoxclusters</a>. BMC Developmental Biology 10(1): 78-92. <strong>Q1, IF: 2.78, IC: 26</strong></p>
<p>Jakó É, <strong>Ari E</strong>, Ittzés P, Horváth A &amp; Podani J (2009) <a href="https://www.sciencedirect.com/science/article/abs/pii/S1055790309001584">BOOL-AN: A method for comparative sequence analysis and phylogenetic reconstruction</a>. Molecular Phylogenetics and Evolution 52(3): 887-897. <strong>D1, IF: 3.88, IC: 29</strong></p>
<p><strong>Ari E</strong>, Ittzés P, Podani J &amp; Jakó É (2008) <a href="https://www.researchgate.net/publication/216330267_Phylogenetic_Tree_Reconstruction_with_a_New_Discrete_Mathematical_Method">Phylogenetic tree reconstruction with a new discrete mathematical method</a>. Kitaibelia 13: 209-211. (in Hungarian) <strong>Q2, IF: 0.23</strong></p>
</section>
</section>
<section id="oral-presentations-in-conferences" class="level1">
<h1><strong>ORAL PRESENTATIONS IN CONFERENCES</strong></h1>
<section id="nr.-of-international-conference-talks-9-nr.-of-national-conference-talks-6-as-an-invited-speaker-3" class="level3">
<h3 class="anchored" data-anchor-id="nr.-of-international-conference-talks-9-nr.-of-national-conference-talks-6-as-an-invited-speaker-3"><strong>Nr. of international conference talks: 9; Nr. of national conference talks: 6; As an invited speaker*: 3</strong></h3>
<p>Ari E, et al.&nbsp;(2024) mulea - an R package for enrichment analysis using multiple ontologies and empirical false discovery rate. <a href="https://indico.wigner.hu/event/1583/overview">SciComp24 Conference</a>, 17-19 October. Szeged, H</p>
<p>Ari E, et al.&nbsp;(2023) Global map of evolutionary dependencies between antibiotic resistance and virulence genes in E. coli. <a href="https://www.hcemm.eu/news/2023/11/3rd-hpps-phd-postdoc-symposium-keszthely/">3rd HCEMM PhD-POSTDOC Symposium</a>, 8-9 November. Keszthely, H</p>
<p>Ari E, et al.&nbsp;(2022) Global map of evolutionary dependencies between antibiotic resistance and virulence genes in E. coli. <a href="https://konferencia.unideb.hu/en/BDG2022">1st Bioinformatics and Data Science in Genomic Studies (BDG2022)</a> online conference, 25 Nov, University of Debrecen, H</p>
<p>Ari E, et al.&nbsp;(2022) Global map of evolutionary dependencies between antibiotic resistance and virulence genes in E. coli. Bioinformatics, Celebrating the Hungarian Science, 11 Nov, HAS, Research Centre for Natural Sciences, Budapest, H</p>
<p>Ari E*, Kintses B (2020) Methods to investigate the microbiome and understand the results. Meeting of Hungarian Society for Gastroenterology, Section Colon, 6-7 March, Visegrád, H</p>
<p>Ari E, et al.&nbsp;(2019) Phylogenetic barriers to horizontal transfer of antimicrobial peptide resistance genes in the human gut microbiota. EvolBiol Day, 17 Apr, Szeged, H</p>
<p>Ari E*, et al.&nbsp;(2018) How well do the antimicrobial peptide resistance genes spread? Bioinformatics, Celebrating the Hungarian Science, 16 Nov, HAS, Research Centre for Natural Sciences, Budapest, H</p>
<p>Ari E: (2017) MulEA – A tool for multi-enrichment analysis, 2nd Interdisciplinary Signaling Workshop, 17-21 July, Visegrád, H</p>
<p>Ari E*: (2012) Molecular phylogenetic reconstructions with a discrete mathematical method, the Boolean analysis. International Congress, Hungarian Society for Microbiology, 24-26 Oct, Keszthely, H</p>
<p>Ari E &amp; Jakó É (2012) Comparison of Boolean analysis and standard phylogenetic methods using artificially evolved and natural mt-tRNA sequences from great apes. 9th Joint Conference on Mathematics and Computer Science, 9-12 Feb, Siófok, H</p>
<p>Ari E, et al.&nbsp;(2007) Boolean analysis: A new discrete mathematical method for phylogenetic reconstruction. From Molecular Informatics to Bioinformatics – International Symposium, 19-21 April, Collegium Budapest, H</p>
<p>Ari E &amp; Jakó É (2007) Testing a new discrete mathematical method for phylogenetic reconstruction. Evolution 2007 Conference, 16-20 June, Christchurch, NZ</p>
<p>Ari E &amp; Jakó É (2007) Testing a new discrete mathematical method for phylogenetic reconstruction. The Dumont D’Urville Workshop on Applied Evolutionary Bioinformatics, 24-27 June, Kaikoura, NZ</p>
<p>Ari E, et al.&nbsp;(2007) Phylogenetic tree reconstruction with a new discrete mathematical method. Molecular taxonomic, phylogenetic and phylogeographic researches in Hungary, Meeting in memory of Sámuel Diószegi, 17 Nov, Debrecen, H</p>
<p>Ari E (2006) Reconstructing the phylogenetic tree of great apes by using a new discrete mathematical method. 12th Annual European Meeting of PhD students in Evolutionary Biology, 4-9 Sep, St.&nbsp;Andrews, UK</p>
</section>
</section>
<section id="other-invited-talks" class="level1">
<h1><strong>OTHER INVITED TALKS</strong></h1>
<p>Ari E* (2017) Investigating the antimicrobial peptide resistome in the human gut microbiome: a metagenomic approach. University of Graz, Institute of Zoology, 15 Nov, Graz, A</p>
<p>Ari E*: (2015) Rapid evolution of phenotypic plasticity during experimental evolution of Drosophila. The Genome Analysis Centre, 23 Feb, Norwich, UK</p>
</section>
<section id="poster-presentations-in-conferences" class="level1">
<h1><strong>POSTER PRESENTATIONS IN CONFERENCES</strong></h1>
<section id="nr.-of-international-conference-posters-9" class="level3">
<h3 class="anchored" data-anchor-id="nr.-of-international-conference-posters-9"><strong>Nr. of international conference posters: 9</strong></h3>
<p>Ari E, et al.&nbsp;(2023) <a href="https://indico.ictp.it/event/10210/material/4/0.pdf">Global map of evolutionary dependencies between antibiotic resistance and virulence genes in E. coli</a>, <a href="https://meetings.embo.org/event/23-amr-plasmids">EMBO Workshop: Plasmids as vehicles of AMR spread</a>, 12-18 September, (Trieste, Italy) online</p>
<p>Ari E, et al.&nbsp;(2023) Global map of evolutionary dependencies between antibiotic resistance and virulence genes in E. coli, <a href="https://www.embl.org/about/info/course-and-conference-office/events/pev23-01/">EMBO Workshop: Predicting evolution</a>, 11-14 July, Heidelberg, Germany</p>
<p>Ari E, et al.&nbsp;(2022) <a href="https://www.lamg.info/_files/ugd/2e355c_d62492daad5840479e27865c3233bfba.pdf">Global map of evolutionary dependencies between antibiotic resistance and virulence genes in E. coli</a>. <a href="https://www.lamg.info/">Lake Arrowhead Microbial Genomics Conference</a>, 11-15 September, Lake Arrowhead, CA, USA</p>
<p>Rutka M, Szántó K, Bacsur P, Resál T, Jójárt B, Bálint A, Ari E, Kintses B, Fehér T, Asbóth A, Pigniczki D, Bor R, Fábián A, Farkas K, Maléth J, Szepes Z &amp; Molnár T (2022) <a href="https://academic.oup.com/ecco-jcc/article/16/Supplement_1/i609/6513059">P713 Gut Microbiota Alterations after Bowel Preparation amongst Inflammatory Bowel Disease Patients</a>. Journal of Crohn’s and Colitis, 16 (Supplement_1): i609. <strong>Q1, IF: 9.07</strong></p>
<p>Ari E, et al.&nbsp;(2019) <a href="https://www.researchgate.net/publication/338231106_Phylogenetic_barriers_to_horizontal_transfer_of_AMP_resistance_genes">Phylogenetic barriers to horizontal transfer of antimicrobial peptide resistance genes in the human gut microbiota</a>. Gordon Research Conference on Molecular Mechanisms in Evolution, 9-14 June, Easton, MA, USA</p>
<p>Ari E, et al.&nbsp;(2019) <a href="https://www.researchgate.net/publication/338231106_Phylogenetic_barriers_to_horizontal_transfer_of_AMP_resistance_genes">Phylogenetic barriers to horizontal transfer of antimicrobial peptide resistance genes in the human gut microbiota</a>. Gordon Research Seminar on Molecular Mechanisms in Evolution, 8-9 June, Easton, MA, USA</p>
<p>Ari E &amp; Jakó É (2015) Graph-based generalized Boolean descriptors for classification of biological macromolecules. Conferentia Chemometrica, 13-16 Sep, Budapest, H</p>
<p>Ari E et al.&nbsp;(2015) Rapid evolution of phenotypic plasticity during experimental evolution of Drosophila. Annual Meeting of the Society for Molecular Biology and Evolution, 12-16 July, Vienna, A</p>
<p>Ari E, et al.&nbsp;(2006) Testing a new discrete mathematical method for reconstructing the great apes phylogeny based on mitochondrial tRNAs. Evolution of Biomolecular Structure, 25-27 May, Vienna, A</p>


</section>
</section>


<div id="quarto-appendix" class="default"><section id="footnotes" class="footnotes footnotes-end-of-document" role="doc-endnotes"><h2 class="anchored quarto-appendix-heading">Footnotes</h2>

<ol>
<li id="fn1"><p>Formal names: Eötvös Loránd Research Network (ELKH); Research Network of the Hungarian Academy of Science (HAS)<a href="#fnref1" class="footnote-back" role="doc-backlink">↩︎</a></p></li>
<li id="fn2"><p>Formal and actual name: University of Veterinary Medicine<a href="#fnref2" class="footnote-back" role="doc-backlink">↩︎</a></p></li>
</ol>
</section></div></main> <!-- /main -->
<script id="quarto-html-after-body" type="application/javascript">
window.document.addEventListener("DOMContentLoaded", function (event) {
  const toggleBodyColorMode = (bsSheetEl) => {
    const mode = bsSheetEl.getAttribute("data-mode");
    const bodyEl = window.document.querySelector("body");
    if (mode === "dark") {
      bodyEl.classList.add("quarto-dark");
      bodyEl.classList.remove("quarto-light");
    } else {
      bodyEl.classList.add("quarto-light");
      bodyEl.classList.remove("quarto-dark");
    }
  }
  const toggleBodyColorPrimary = () => {
    const bsSheetEl = window.document.querySelector("link#quarto-bootstrap");
    if (bsSheetEl) {
      toggleBodyColorMode(bsSheetEl);
    }
  }
  toggleBodyColorPrimary();  
  const disableStylesheet = (stylesheets) => {
    for (let i=0; i < stylesheets.length; i++) {
      const stylesheet = stylesheets[i];
      stylesheet.rel = 'prefetch';
    }
  }
  const enableStylesheet = (stylesheets) => {
    for (let i=0; i < stylesheets.length; i++) {
      const stylesheet = stylesheets[i];
      stylesheet.rel = 'stylesheet';
    }
  }
  const manageTransitions = (selector, allowTransitions) => {
    const els = window.document.querySelectorAll(selector);
    for (let i=0; i < els.length; i++) {
      const el = els[i];
      if (allowTransitions) {
        el.classList.remove('notransition');
      } else {
        el.classList.add('notransition');
      }
    }
  }
  const toggleColorMode = (alternate) => {
    // Switch the stylesheets
    const alternateStylesheets = window.document.querySelectorAll('link.quarto-color-scheme.quarto-color-alternate');
    manageTransitions('#quarto-margin-sidebar .nav-link', false);
    if (alternate) {
      enableStylesheet(alternateStylesheets);
      for (const sheetNode of alternateStylesheets) {
        if (sheetNode.id === "quarto-bootstrap") {
          toggleBodyColorMode(sheetNode);
        }
      }
    } else {
      disableStylesheet(alternateStylesheets);
      toggleBodyColorPrimary();
    }
    manageTransitions('#quarto-margin-sidebar .nav-link', true);
    // Switch the toggles
    const toggles = window.document.querySelectorAll('.quarto-color-scheme-toggle');
    for (let i=0; i < toggles.length; i++) {
      const toggle = toggles[i];
      if (toggle) {
        if (alternate) {
          toggle.classList.add("alternate");     
        } else {
          toggle.classList.remove("alternate");
        }
      }
    }
    // Hack to workaround the fact that safari doesn't
    // properly recolor the scrollbar when toggling (#1455)
    if (navigator.userAgent.indexOf('Safari') > 0 && navigator.userAgent.indexOf('Chrome') == -1) {
      manageTransitions("body", false);
      window.scrollTo(0, 1);
      setTimeout(() => {
        window.scrollTo(0, 0);
        manageTransitions("body", true);
      }, 40);  
    }
  }
  const isFileUrl = () => { 
    return window.location.protocol === 'file:';
  }
  const hasAlternateSentinel = () => {  
    let styleSentinel = getColorSchemeSentinel();
    if (styleSentinel !== null) {
      return styleSentinel === "alternate";
    } else {
      return false;
    }
  }
  const setStyleSentinel = (alternate) => {
    const value = alternate ? "alternate" : "default";
    if (!isFileUrl()) {
      window.localStorage.setItem("quarto-color-scheme", value);
    } else {
      localAlternateSentinel = value;
    }
  }
  const getColorSchemeSentinel = () => {
    if (!isFileUrl()) {
      const storageValue = window.localStorage.getItem("quarto-color-scheme");
      return storageValue != null ? storageValue : localAlternateSentinel;
    } else {
      return localAlternateSentinel;
    }
  }
  let localAlternateSentinel = 'default';
  // Dark / light mode switch
  window.quartoToggleColorScheme = () => {
    // Read the current dark / light value 
    let toAlternate = !hasAlternateSentinel();
    toggleColorMode(toAlternate);
    setStyleSentinel(toAlternate);
  };
  // Ensure there is a toggle, if there isn't float one in the top right
  if (window.document.querySelector('.quarto-color-scheme-toggle') === null) {
    const a = window.document.createElement('a');
    a.classList.add('top-right');
    a.classList.add('quarto-color-scheme-toggle');
    a.href = "";
    a.onclick = function() { try { window.quartoToggleColorScheme(); } catch {} return false; };
    const i = window.document.createElement("i");
    i.classList.add('bi');
    a.appendChild(i);
    window.document.body.appendChild(a);
  }
  // Switch to dark mode if need be
  if (hasAlternateSentinel()) {
    toggleColorMode(true);
  } else {
    toggleColorMode(false);
  }
  const icon = "";
  const anchorJS = new window.AnchorJS();
  anchorJS.options = {
    placement: 'right',
    icon: icon
  };
  anchorJS.add('.anchored');
  const isCodeAnnotation = (el) => {
    for (const clz of el.classList) {
      if (clz.startsWith('code-annotation-')) {                     
        return true;
      }
    }
    return false;
  }
  const clipboard = new window.ClipboardJS('.code-copy-button', {
    text: function(trigger) {
      const codeEl = trigger.previousElementSibling.cloneNode(true);
      for (const childEl of codeEl.children) {
        if (isCodeAnnotation(childEl)) {
          childEl.remove();
        }
      }
      return codeEl.innerText;
    }
  });
  clipboard.on('success', function(e) {
    // button target
    const button = e.trigger;
    // don't keep focus
    button.blur();
    // flash "checked"
    button.classList.add('code-copy-button-checked');
    var currentTitle = button.getAttribute("title");
    button.setAttribute("title", "Copied!");
    let tooltip;
    if (window.bootstrap) {
      button.setAttribute("data-bs-toggle", "tooltip");
      button.setAttribute("data-bs-placement", "left");
      button.setAttribute("data-bs-title", "Copied!");
      tooltip = new bootstrap.Tooltip(button, 
        { trigger: "manual", 
          customClass: "code-copy-button-tooltip",
          offset: [0, -8]});
      tooltip.show();    
    }
    setTimeout(function() {
      if (tooltip) {
        tooltip.hide();
        button.removeAttribute("data-bs-title");
        button.removeAttribute("data-bs-toggle");
        button.removeAttribute("data-bs-placement");
      }
      button.setAttribute("title", currentTitle);
      button.classList.remove('code-copy-button-checked');
    }, 1000);
    // clear code selection
    e.clearSelection();
  });
  function tippyHover(el, contentFn) {
    const config = {
      allowHTML: true,
      content: contentFn,
      maxWidth: 500,
      delay: 100,
      arrow: false,
      appendTo: function(el) {
          return el.parentElement;
      },
      interactive: true,
      interactiveBorder: 10,
      theme: 'quarto',
      placement: 'bottom-start'
    };
    window.tippy(el, config); 
  }
  const noterefs = window.document.querySelectorAll('a[role="doc-noteref"]');
  for (var i=0; i<noterefs.length; i++) {
    const ref = noterefs[i];
    tippyHover(ref, function() {
      // use id or data attribute instead here
      let href = ref.getAttribute('data-footnote-href') || ref.getAttribute('href');
      try { href = new URL(href).hash; } catch {}
      const id = href.replace(/^#\/?/, "");
      const note = window.document.getElementById(id);
      return note.innerHTML;
    });
  }
      let selectedAnnoteEl;
      const selectorForAnnotation = ( cell, annotation) => {
        let cellAttr = 'data-code-cell="' + cell + '"';
        let lineAttr = 'data-code-annotation="' +  annotation + '"';
        const selector = 'span[' + cellAttr + '][' + lineAttr + ']';
        return selector;
      }
      const selectCodeLines = (annoteEl) => {
        const doc = window.document;
        const targetCell = annoteEl.getAttribute("data-target-cell");
        const targetAnnotation = annoteEl.getAttribute("data-target-annotation");
        const annoteSpan = window.document.querySelector(selectorForAnnotation(targetCell, targetAnnotation));
        const lines = annoteSpan.getAttribute("data-code-lines").split(",");
        const lineIds = lines.map((line) => {
          return targetCell + "-" + line;
        })
        let top = null;
        let height = null;
        let parent = null;
        if (lineIds.length > 0) {
            //compute the position of the single el (top and bottom and make a div)
            const el = window.document.getElementById(lineIds[0]);
            top = el.offsetTop;
            height = el.offsetHeight;
            parent = el.parentElement.parentElement;
          if (lineIds.length > 1) {
            const lastEl = window.document.getElementById(lineIds[lineIds.length - 1]);
            const bottom = lastEl.offsetTop + lastEl.offsetHeight;
            height = bottom - top;
          }
          if (top !== null && height !== null && parent !== null) {
            // cook up a div (if necessary) and position it 
            let div = window.document.getElementById("code-annotation-line-highlight");
            if (div === null) {
              div = window.document.createElement("div");
              div.setAttribute("id", "code-annotation-line-highlight");
              div.style.position = 'absolute';
              parent.appendChild(div);
            }
            div.style.top = top - 2 + "px";
            div.style.height = height + 4 + "px";
            let gutterDiv = window.document.getElementById("code-annotation-line-highlight-gutter");
            if (gutterDiv === null) {
              gutterDiv = window.document.createElement("div");
              gutterDiv.setAttribute("id", "code-annotation-line-highlight-gutter");
              gutterDiv.style.position = 'absolute';
              const codeCell = window.document.getElementById(targetCell);
              const gutter = codeCell.querySelector('.code-annotation-gutter');
              gutter.appendChild(gutterDiv);
            }
            gutterDiv.style.top = top - 2 + "px";
            gutterDiv.style.height = height + 4 + "px";
          }
          selectedAnnoteEl = annoteEl;
        }
      };
      const unselectCodeLines = () => {
        const elementsIds = ["code-annotation-line-highlight", "code-annotation-line-highlight-gutter"];
        elementsIds.forEach((elId) => {
          const div = window.document.getElementById(elId);
          if (div) {
            div.remove();
          }
        });
        selectedAnnoteEl = undefined;
      };
      // Attach click handler to the DT
      const annoteDls = window.document.querySelectorAll('dt[data-target-cell]');
      for (const annoteDlNode of annoteDls) {
        annoteDlNode.addEventListener('click', (event) => {
          const clickedEl = event.target;
          if (clickedEl !== selectedAnnoteEl) {
            unselectCodeLines();
            const activeEl = window.document.querySelector('dt[data-target-cell].code-annotation-active');
            if (activeEl) {
              activeEl.classList.remove('code-annotation-active');
            }
            selectCodeLines(clickedEl);
            clickedEl.classList.add('code-annotation-active');
          } else {
            // Unselect the line
            unselectCodeLines();
            clickedEl.classList.remove('code-annotation-active');
          }
        });
      }
  const findCites = (el) => {
    const parentEl = el.parentElement;
    if (parentEl) {
      const cites = parentEl.dataset.cites;
      if (cites) {
        return {
          el,
          cites: cites.split(' ')
        };
      } else {
        return findCites(el.parentElement)
      }
    } else {
      return undefined;
    }
  };
  var bibliorefs = window.document.querySelectorAll('a[role="doc-biblioref"]');
  for (var i=0; i<bibliorefs.length; i++) {
    const ref = bibliorefs[i];
    const citeInfo = findCites(ref);
    if (citeInfo) {
      tippyHover(citeInfo.el, function() {
        var popup = window.document.createElement('div');
        citeInfo.cites.forEach(function(cite) {
          var citeDiv = window.document.createElement('div');
          citeDiv.classList.add('hanging-indent');
          citeDiv.classList.add('csl-entry');
          var biblioDiv = window.document.getElementById('ref-' + cite);
          if (biblioDiv) {
            citeDiv.innerHTML = biblioDiv.innerHTML;
          }
          popup.appendChild(citeDiv);
        });
        return popup.innerHTML;
      });
    }
  }
    var localhostRegex = new RegExp(/^(?:http|https):\/\/localhost\:?[0-9]*\//);
      var filterRegex = new RegExp('/' + window.location.host + '/');
    var isInternal = (href) => {
        return filterRegex.test(href) || localhostRegex.test(href);
    }
    // Inspect non-navigation links and adorn them if external
 	var links = window.document.querySelectorAll('a[href]:not(.nav-link):not(.navbar-brand):not(.toc-action):not(.sidebar-link):not(.sidebar-item-toggle):not(.pagination-link):not(.no-external):not([aria-hidden]):not(.dropdown-item)');
    for (var i=0; i<links.length; i++) {
      const link = links[i];
      if (!isInternal(link.href)) {
          // target, if specified
          link.setAttribute("target", "_blank");
      }
    }
});
</script>
</div> <!-- /content -->



</body></html>